content/en/agent/guide/python-3.md

```diff
@@ -9,6 +9,31 @@ This guide provides information and best practices on migrating checks between P
 
 To provide flexibility in allowing code to run multiple on versions of the Agent, this guide focuses on retaining backwards compatibility.
 
+## Agent configuration
+
+Starting with v6.14.0, the Agent integrates both Python 2 and Python 3 runtimes. This means that checks can be run with either Python 2 or Python 3, depending on the Agent configuration.
+
+By default, the Agent v6 will use the Python 2 runtime. To switch to the Python 3 runtime, set the `python_version` configuration option in the `datadog.yaml` configuration file:
+
+```yaml
+python_version: 3
+```
+
+and restart the Agent.
+
+Alternatively, the `DD_PYTHON_VERSION` environment variable can be set to `2` or `3` to choose which Python runtime is used. If it is set, the `python_version` option in `datadog.yaml` will be ignored.
```

Comment:
This new env variable should be documented here: https://docs.datadoghq.com/agent/docker/?tab=standard#environment-variables as well.

Reply:
I don't think this option should be documented in the Docker Agent's pages, since it should *not* be used with the Containerized Agent (as mentioned in the next section, you choose your python version by choosing a different Docker Agent image depending on whether you use Python 2 or Python 3 ; using the `DD_PYTHON_VERSION` option with the Docker Agent could actually be harmful, since you could set the option to a Python version that is not in the image, which would prevent any Python check from running).

I do think however that we could add a link to the Containerized Agent section of the Python 3 page in the Docker Agent's page, so that people reading the Docker Agent doc can easily know how to have an image running Python 3.